src/main/java/dev/esophose/playerparticles/particles/spawning/ParticleSpawner.java

```diff
@@ -136,9 +137,9 @@ public static boolean canSee(Player player, Player target) {
         if (player == null || target == null)
             return true;
 
-        for (MetadataValue meta : target.getMetadata("vanished"))
+        /*for (MetadataValue meta : target.getMetadata("vanished"))
```

Comment:
Why was this removed? It breaks vanish plugin support.

Reply:
Read the comments for each commit, it has bad performance implications + if a player is hidden, player.canSee(target) will return false anyways.

Reply:
Didn't see there were comments, go ahead and keep it as-is then. I'll rework this to run on the same task as the WorldGuard region checks so it's less performance heavy.